tests/lib/python/test_diffr.py

```diff
@@ -0,0 +1,414 @@
+import json
+import os
+from subprocess import Popen, PIPE
+from textwrap import dedent
+import unittest
+
+from diffr import Diff, __file__ as SCRIPT
+
+
+class TestDiff(unittest.TestCase):
+
```

Comment:
Probably ought to have a test to ensure that sensible error is raised if int, str, float or any other non iterable is fed to it.